src/services/user/firstLogin.js

```diff
@@ -100,7 +100,7 @@ const firstLogin = async (data, params, app) => {
 			return consent;
 		};
 
-		updateConsentUsingVersions = app.service('consents').find({ query: { userId: user._id } }).then((consents) => {
+		updateConsentUsingVersions = app.service('consents/model').find({ query: { userId: user._id } }).then((consents) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 125. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5276619)